code/Data_Analysis.ipynb

```diff
@@ -972,7 +972,7 @@
    "id": "747eaa15",
    "metadata": {},
    "source": [
-    "In the next step, 'cross_val_predict' is used to generate predictions each iteration. 'LinearSCV' will be set as estimator."
+    "In the next step, 'cross_val_predict' is used to generate predictions each iteration. 'LinearSCV' will be set as estimator. It stands for Linear Support Vector Classification and predicts discrete labels for each sample using a \"one-vs-the-rest\" multi-class strategy."
```

Comment:
Could you maybe add a link to the `LinearSVC` docs and further information regarding the model itself (it's a linear model, SVM, multi-class strategy, etc.)?